index.html

```diff
@@ -1,13 +1,139 @@
+    <title>Web Summit</title>
 </head>
+
 <body>
-    
+    <!--Header Section-->
+    <header id="header">
+        <ul id="header-list">
+            <li class="header-item icon"><a href=""><i class="fa fa-facebook"></i></a></li>
+            <li class="header-item icon"><a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+            <li class="header-item"><a href="">English</a></li>
+            <li class="header-item"><a href="">My Page</a></li>
+            <li class="header-item"><a href="">Logout</a></li>
+        </ul>
+    </header>
+
+    <!--Navbar Section-->
+    <nav id="navbar">
+        <a class="logo" href="index.html"><img src="images/logo.png" alt="Web Submmit"></a>
+        <ul class="nav-list">
+            <li class="nav-item"><a href="about.html">About</a></li>
+            <li class="nav-item link"><a href="#main-programs">Program</a></li>
+            <li class="nav-item link"><a href="#speakers-section">Speakers</a></li>
+            <li class="nav-item link"><a href="#sponsors-section">Sponsors</a></li>
+            <li class="nav-item button link"><a href="">GC Campaign</a></li>
+            <li id="close-times"><a id="close" href=""><i class="fa fa-times"></i></a></li>
+        </ul>
+
+        <div id="mobile-menue">
+            <i id="bar" class="fa fa-bars"></i>
+        </div>
+
+    </nav>
+
+    <!--Main Section-->
+    <main id="main">
+
+        <!-- Headline Section -->
+        <section id="headline">
+            <h3>"Hello Web Lovers!"</h3>
+            <h1>Web Technology <br> Global Submmit <br> 2022</h1>
+            <p id="details">Web Summit 2022 will bring together 70,000+ people, and the companies redefining the tech
+                industry.
+                Incredible speakers, unparalleled networking opportunities, and proprietary software that will maximise
+                your experience.
+            </p>
+            <h2 id="time">12.05.2022(THU) ~ 14(SAT)</h2>
+            <p id="place">@ Boston Convention and Exhibition Center Boston, MA</p>
+        </section>
+
+        <!-- Program Section -->
+        <section id="main-programs">
+            <h3>Main Program</h3>
+            <div class="section-underline"></div>
+            <div id="programs">
+                <div class="program">
+                    <img src="images/lecture.png" alt="Lecture">
+                    <h4>Lecture</h4>
+                    <p>Listen to speakers from various countries and from different backgrounds talk about their latest
+                        experiences and creations.</p>
+                </div>
+
+                <div class="program">
+                    <img src="images/stage.png" alt="Lecture">
+                    <h4>WS Exhibition</h4>
+                    <p>Appreciate various creations inluding new web Technologies.</p>
+                </div>
+
+                <div class="program">
+                    <img src="images/forum.png" alt="Lecture">
```

Comment:
- [ ] Please ensure that the program section cards are all same size as  it is in the original design.

#### Your Design
![image](https://user-images.githubusercontent.com/43995830/170722046-b2112d71-ef34-423c-991d-996e33650725.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/170721876-58942664-1508-4917-98a2-6180722db298.png)